drivers/dp/Kconfig

```diff
@@ -24,7 +24,18 @@ config SWDP_BITBANG_DRIVER
 	bool "Serial Wire Debug Port bit-bang driver"
 	default y
 	depends on DT_HAS_ZEPHYR_SWDP_GPIO_ENABLED
+	depends on GPIO
 	help
 	  Serial Wire Debug Port bit-bang driver.
 
+if SWDP_BITBANG_DRIVER
+
+config SWDP_BITBANG_DRIVER_SEPARATE_IN_OUT
+	bool "SWDIO is connected through transceiver"
+	default n
+	help
+	  Instead of DIO, DIN, DOUT and DNOE should be defined.
```

Comment:
Driver supports multiples instances, therefore this cannot be a Kconfig option.

Reply:
Could you show me a way how to implement this only using device tree?
Also, I'm not sure if I see a use case where you'd have one instance with and another instance without separate I/O lines.

Reply:
> Also, I'm not sure if I see a use case where you'd have one instance with and another instance without separate I/O lines.

Well it is now we design the drivers, and Kconfig is not correct approach for this option. I can fix it for you.

Reply:
Please do.